h2o-r/h2o-package/R/frame.R

```diff
@@ -2170,6 +2170,7 @@ h2o.nlevels <- function(x) {
 #' that are referencing this column). If you want to make a copy of the column instead, use parameter in.place = FALSE.
 #'
 #' @param x A single categorical column.
+#' @param destination_frame A string indicating the destination key. If empty, this will be auto-generated by H2O.
```

Comment:
??
`destination_frame` is not a parameter in setLevels. It's the inconsistency use of `new_levels`/`new.levels` in the example that is causing the error.